db.js

```diff
@@ -774,7 +879,6 @@ exports.init = function (sbot, config) {
     getState: () => state,
     getIndexes: () => indexes,
     getIndex: (index) => indexes[index],
-    clearIndexes,
```

Comment:
This was something I used in browser core for testing. I don't mind removing this. It's a thin wrapper around indexes reset anyway.

Reply:
Do you want to bring it back?